app/controllers/api/teams.php

```diff
@@ -42,11 +45,7 @@
     ->inject('user')
     ->inject('dbForProject')
     ->inject('events')
-    ->action(function ($teamId, $name, $roles, $response, $user, $dbForProject, $events) {
-        /** @var Appwrite\Utopia\Response $response */
-        /** @var Utopia\Database\Document $user */
-        /** @var Utopia\Database\Database $dbForProject */
-        /** @var Appwrite\Event\Event $events */
+    ->action(function ($teamId, $name, $roles, Response $response, Document $user, Database $dbForProject, Event $events) {
```

Comment:
```suggestion
    ->action(function (string $teamId, string $name, array $roles, Response $response, Document $user, Database $dbForProject, Event $events) {
```

I think we can also add primitive types to params we get from the http server.

Reply:
Sure ! I'll make the changes